fxgl-core/src/main/java/com/almasb/fxgl/core/math/FXGLMath.java

```diff
@@ -383,11 +383,11 @@ public static Point2D bezier(Point2D p1, Point2D p2, Point2D p3, Point2D p4, dou
     }
 
     /**
-     * @param t current time * frequency (lower frequency -> smoother output)
-     * @return perlin noise in 1D quality in [0..1)
+     * @param seed used to create unique noise generator with replicable output
```

Comment:
Sorry if I wasn't clear before. We should have both:

```
noise1D(t)

and

getNoise1DGenerator(long seed)
```

FXGLMath should have a `private static final generator` to produce the 1D values:

```
noise1D(t) { return generator.noise1D(t) }
```